sirenity/code/handler.py

```diff
@@ -0,0 +1,43 @@
+import ast
+import os
+
+allowed_modules = ["math"]
+
+
+class CodeHandler:
+    """A class used to handle code submissions from users"""
+
+    pass
+
+    def __init__(self, code: str) -> None:
+        self.code = code
+        pass
+
+    def safe_run(self):
+        """Safetly runs python code"""
+        tree = ast.parse(self.code)
+        safe_nodes = self.get_safe_nodes()
+        for node in tree.body:
+            node_name = type(node).__name__
+            if node_name == "Import":
```

Comment:
```suggestion
            if isinstance(node, ast.Import):
```
See if this works instead